### DISCUSSION TITLE
The reference.docx is not online

### DISCUSSION THREAD

**Discussion by aburkov** [jgm/pandoc#9772] [Q&A] [answered]
Hi! The [documentation](https://pandoc.org/MANUAL.html) suggests "To produce a custom reference.docx, first get a copy of the default reference.docx". I don't need a custom reference.docx, I just need the default one, but after an hour of intense googling, I only found an unofficial file [here](https://github.com/hokorobi/pandoc-docx-reference/blob/master/reference.docx).

Where can I get the file mentioned in the official doc?

**Comment by jgm** [comment, marked as answer]
You just needed to read a bit further in the documentation:

>  first get a copy of the default reference.docx: `pandoc -o custom-reference.docx --print-default-data-file reference.docx`. 

That command line tells you how to produce a copy of it using pandoc.

**Reply by aburkov** [reply]
I didn't expect that. Thank you!

**Reply by aburkov** [reply]
Oh, by the way, Word doesn't allow modifying the alignment in the Verbatim Char style:
<img width="642" alt="Screenshot 2024-05-18 at 11 08 58 PM" src="https://github.com/jgm/pandoc/assets/12722258/ca209e78-566a-4e33-ad68-6ce536dd0946">
Do you know how to do that?

I tried to find Caption Char, the style Verbatim Char is based on, with no luck.

**Reply by jgm** [reply]
Alignments don't make sense for character styles.  Try modifying SourceCode.
